xbmc/GUIInfoManager.cpp

```diff
@@ -2323,7 +2323,13 @@ const infomap mediacontainer[] = {{ "hasfiles",         CONTAINER_HASFILES },
 ///   \table_row3{   <b>`Container(id).NumItems`</b>,
 ///                  \anchor Container_NumItems
 ///                  _boolean_,
-///     Number of items in the container or grouplist with given id. If no id is
+///     Number of items in the container or grouplist with given id, excluding parent folder item. If no id is
+///     specified it grabs the current container.
+///   }
+///   \table_row3{   <b>`Container(id).ActualItems`</b>,
+///                  \anchor Container_ActualItems
+///                  _boolean_,
+///     Number of items in the container or grouplist with given id, including parent folder item. If no id is
```

Comment:
Don't forget the `\` before the commas in text!

Is the bad part on doxygen if on a Macro (e.g. `\table_row3`) are more as one value, then the comma separate the values. If now a text is there also with them brings it problems.